binder/environment.yml

```
@@ -0,0 +1,6 @@
+name: base
+channels:
+  - conda-forge
+dependencies:
+  - numpy
+  - beakerx
```

To fix the Java version issue:
```suggestion
  - beakerx
  - openjdk
```